modules/serverlist.js

```diff
@@ -0,0 +1,81 @@
+// Made by epsilon#3892
+// https://forum.thd.vg/members/16800/
+
+var config = require("../config.json");
+var request = require("request")
+
+module.exports = function(client) {
+	var module = {};
```

Comment:
you're not using this variable

Reply:
im using it to fetch the message (line 11)

Reply:
I'm talking about the module variable

Reply:
don't i need that for when it is returned at the end? ive only just learnt how to use modules through this bot